### ISSUE TITLE
Issues with SSL certificate (?) while generating tokenList.json

### ISSUE THREAD

**Issue by drptbl** [CityOfZion/neo-tokens#38]
**Details:**
`master` branch (bca6974)

**Environment:**
Win 10 Home (64 bit) Version 1803 Build 17134.112

**Reproduction steps:**
1. Run `yarn`
2. Run `yarn start`

**Console error:**
```
test@test MINGW64 ~/source/neo-tokens (master)
$ yarn start
yarn run v1.9.4
$ node --harmony index.js
(node:18608) UnhandledPromiseRejectionWarning: FetchError: request to https://n1.cityofzion.io/v1/tokens failed, reason: write EPROTO 101057795:error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake fai
lure:openssl\ssl\s23_clnt.c:802:

    at ClientRequest.<anonymous> (C:\source\neo-tokens\node_modules\node-fetch\index.js:133:11)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at TLSSocket.socketErrorListener (_http_client.js:387:9)
    at emitOne (events.js:116:13)
    at TLSSocket.emit (events.js:211:7)
    at onwriteError (_stream_writable.js:418:12)
    at onwrite (_stream_writable.js:440:5)
    at _destroy (internal/streams/destroy.js:39:7)
    at TLSSocket.Socket._destroy (net.js:564:3)
(node:18608) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with
 .catch(). (rejection id: 14)
(node:18608) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 2.87s.
```

**Comment by drptbl** [comment]
Confirmed that the same issue exists on MacOS.

**Comment by drptbl** [comment]
For future reference:
It seems that it's an issue related to node >= 8.6.0 only (https://github.com/nodejs/node/issues/16196).
Tested this with latest node (8.12.0) and it doesn't seem to work properly.

A workaround to make it work is to add this inline fix:
`require('tls').DEFAULT_ECDH_CURVE = 'auto'`

**Comment by drptbl** [comment]
Closing, should be fixed with #39.
